lib/ops/version.rb

```diff
@@ -1,3 +1,3 @@
 module Ops
-  VERSION = '0.5.2'
+  VERSION = '0.5.3'
```

Comment:
> PATCH version when you make backwards-compatible bug fixes.

if `:repo_name` is not specified, it defaults to original behavior, making it backwards compatible and optional. 

Am I reading into this the wrong way?